### ISSUE TITLE
Index.html

### ISSUE THREAD

**Issue by josephgupkim** [jfromak/prj-rev-bwfs-dasmoto#1]
The elements and tags existing are great but the consistency with the code structure could use some work.

Remember that it is crucial to use indentations and whitespace correctly!
Using these concepts result in an organized code structure that aids in legibility for both you and others reading your code.
It also helps see the parent/child relationship between relevant tags, as well as where our tags/content begin and end. This is important for visualizing concepts related to Flexbox which you will learn in the weeks to come.
I hope that for future projects you will use proper indentation procedures to organize your code!

**Comment by PradnyaBora** [comment]
Hi Joseph,

I want to work on his this issue. From above description I understood indentation required. So should I start with that.

Thanks,
Pradnya Bora

**Comment by josephgupkim** [comment]
Yes! Please do start with indentations. It's simple concept that is VERY important as a developer.

Here is simple article to get you started:
https://www.granneman.com/webdev/coding/formatting-and-indenting-your-html/
